### ISSUE TITLE
Use official django-customfield again, or remove it

### ISSUE THREAD

**Issue by valentijnscholten** [DefectDojo/django-DefectDojo#3097]
DD is using its own fork of djang-customfield. It looks like we did only one commit to make it compatible with django 2.x?

The official version now has support for django 2, so could we switch back to that and stay up-to-date with bugfixes, etc?

https://github.com/willseward/django-custom-field

**Comment by valentijnscholten** [comment]
wdyt @Maffooch

**Comment by valentijnscholten** [comment]
or we could remove it maybe as it looks like the `CustomFieldValue` stuff has been replaced by `Dojo_Meta`.

**Comment by Maffooch** [comment]
Works for me. I remember the build failed without making those changes. This was almost two years ago so a lot has changed.

**Comment by dan-kennedy** [comment]
Apologies if this isn't directly related but seeing the following on RHEL 7.9 & trying to figure out how to fix:

`Downloading django-auditlog-0.4.7.tar.gz (11 kB)
Collecting django-custom-field
  Cloning https://github.com/DefectDojo/django-custom-field.git (to revision 4e095dd9119858e90fd21560e5618c6347e4ef1b) to /tmp/pip-wheel-thf29ykk/django-custom-field
ERROR: Command errored out with exit status 128: git clone -q https://github.com/DefectDojo/django-custom-field.git /tmp/pip-wheel-thf29ykk/django-custom-field Check the logs for full command output.
WARNING: You are using pip version 20.2.3; however, version 20.3.3 is available.
You should consider upgrading via the '/usr/local/bin/python -m pip install --upgrade pip' command.
ERROR: Service 'uwsgi' failed to build : The command '/bin/sh -c pip3 wheel --wheel-dir=/tmp/wheels -r ./requirements.txt' returned a non-zero code: 1`

> pip 20.3.3 from /usr/lib/python2.7/site-packages/pip (python 2.7)
> pip 9.0.3 from /usr/lib/python3.6/site-packages (python 3.6)

**Comment by valentijnscholten** [comment]
update to 1.11.0

**Comment by dan-kennedy** [comment]
> update to 1.11.0

Thanks.  I did a fresh clone this morning so it should be on 1.11.0 regardless.  When I try to just do individual builds against uwsgi or nginx, it still errors at the same point which refers to the aforementioned error.

**Comment by valentijnscholten** [comment]
On second thought it doesn't look related to the pip thing. Are you using Docker? Anything is not tested. Also looks like mixing python2 and 3? It looks like the git clone failed.

**Comment by valentijnscholten** [comment]
You could try to install the official customfields instead of our clone, will probably work just fine.

**Comment by dan-kennedy** [comment]
> On second thought it doesn't look related to the pip thing. Are you using Docker? Anything is not tested. Also looks like mixing python2 and 3? It looks like the git clone failed.

Yes this is being done via 'docker-compose build'.  I've tested the clone for djagno-custom-field manually, even into /tmp/ where the script defaults & that works fine.

I have also tried just installing the official manually via PIP & that was fine also: 
"Requirement already satisfied: django-custom-field in /usr/lib/python2.7/site-packages (3.0)"

`$ git clone -q https://github.com/DefectDojo/django-custom-field.git /tmp/pip-wheel-dn2hbwzs/django-custom-field
$ ls -laF /tmp/pip-wheel-dn2hbwzs/
total 12
drwxr-x---.  3 p_dkennedy domain users 4096 Dec 30 11:11 ./
drwxrwxrwt. 17 root       root         4096 Dec 30 11:12 ../
drwxr-x---.  5 p_dkennedy domain users 4096 Dec 30 11:11 django-custom-field/
$ ls -laF /tmp/pip-wheel-dn2hbwzs/django-custom-field/
total 60
drwxr-x---. 5 p_dkennedy domain users 4096 Dec 30 11:11 ./
drwxr-x---. 3 p_dkennedy domain users 4096 Dec 30 11:11 ../
drwxr-x---. 4 p_dkennedy domain users 4096 Dec 30 11:11 custom_field/
drwxr-x---. 8 p_dkennedy domain users 4096 Dec 30 11:11 .git/
-rw-r-----. 1 p_dkennedy domain users  232 Dec 30 11:11 .gitignore
-rw-r-----. 1 p_dkennedy domain users 1067 Dec 30 11:11 LICENSE
-rwxr-x---. 1 p_dkennedy domain users  247 Dec 30 11:11 manage.py*
-rw-r-----. 1 p_dkennedy domain users   43 Dec 30 11:11 MANIFEST.in
-rw-r-----. 1 p_dkennedy domain users 3372 Dec 30 11:11 README.md
drwxr-x---. 2 p_dkennedy domain users 4096 Dec 30 11:11 screencaps/
-rw-r-----. 1 p_dkennedy domain users  924 Dec 30 11:11 setup.py
-rw-r-----. 1 p_dkennedy domain users 1465 Dec 30 11:11 test_settings.py
-rw-r-----. 1 p_dkennedy domain users  163 Dec 30 11:11 test_urls.py
-rw-r-----. 1 p_dkennedy domain users  216 Dec 30 11:11 tox.ini
-rw-r-----. 1 p_dkennedy domain users  252 Dec 30 11:11 .travis.yml`

Pretty odd.  If this wasn't a managed RHEL VM I'd just use the bash setup despite it going EOL tomorrow.  Thanks for your thoughts.  I'll keep attacking this on my end to see what I can sort out.  Happy New Year!
